docs/pages/blog/turbo-1-2-0.mdx

```diff
@@ -0,0 +1,108 @@
+
+You can use multiple filters together to get even more granular filtering as well as combine each part of the above patterns `{}`, `[]` , `^` , and `...` to express more complex behavior.
+
+For example, assuming you have an app located in `./apps/web` directory, some local packages that it uses as dependencies, and a Turborepo pipeline where `test` depends on `^build` topologically, running:
+
+```jsx
+turbo run test --filter={./apps/web}[HEAD^1]^...
+```
+
+would tell `turbo` to ensure that dependencies are built and to run the `test` script in all of the local dependencies of the app located in `./apps/web`, not including that app itself, if the app has changed since HEAD^1.
+
+We believe `--filter` is more expressive and intuitive than `--scope`. In the future, we will be slowly deprecating the `--scope` flag.
+
+For more details and examples, refer to the new [filtering documentation.](/docs/features/filter)
+
+## Debug and Automate with `--dry-run`
+
+You can now see the impact of `turbo run` without actually executing any commands by appending either `--dry-run` or `--dry-run=json` to any `turbo run` command. This will result in either human or JSON output.
+
+Dry runs are incredibly useful for two main situations:
+
+- Debugging and testing run options
+- Using `turbo` filtering and task graphs for building automations
+
+![Turborepo Dry Run](/images/blog/turbo-1-2-0/turbo-dry-run.png)
+
+We hope that this will assist with enhancing observability and debugging, as well as make it easier to leverage `turbo` in dynamic CI/CD systems.
+
+## Improved internal scheduler and graph
+
+When using `turbo run`, every `package.json` task is added to an internal graph to map dependencies based on the inferred relationships defined in your Turborepo `pipeline`. This task graph allows Turborepo to efficiently schedule incremental concurrent task running and cache task outputs for later use.
+
+We've made major improvements to the internal task scheduler and resulting graph structure, resulting in better performance and a better developer experience. For example, in many cases, you will no longer need to use `--include-dependencies`. Instead, after specifying your task entry points, the new and improved graph will automatically handle this graph resolution for you.
+
+## Cache Outputs Integrity and Signature Verification
+
+You can now enable Turborepo to sign remote cache outputs using HMAC-SHA256 with a secret key before uploading them to the Remote Cache. When signed cache artifacts are downloaded, Turborepo will now verify the artifact's integrity and authenticity. Any artifact that fails to verify will be ignored, discarded, and treated as a cache miss by Turborepo.
+
+To enable this feature, set the `remoteCache` options on your `turbo.json` config file to include `signature: true`. Then specify your secret key by declaring the `TURBO_REMOTE_CACHE_SIGNATURE_KEY` environment variable.
```

Comment:
Ooo, framed it differently and now `enable` works. ("enable the feature vs. "enable Turborepo")

```diff
- options on your
+ options in your
```